src/Microsoft.AspNetCore.StaticFiles/StaticFileResponseContext.cs

```diff
@@ -11,14 +12,28 @@ namespace Microsoft.AspNetCore.StaticFiles
     /// </summary>
     public class StaticFileResponseContext
     {
+        public StaticFileResponseContext(HttpContext context, IFileInfo file)
```

Comment:
This is closer to the pattern we use in other repos like Security. Adding back the default constructor will avoid a binary break and avoid getting flagged by our breaking change tools (even if we didn't expect anyone to call that API).

Reply:
default added back in